client/app/models/ImageModel.js

```diff
@@ -1,21 +1,32 @@
 import Immutable from 'immutable';
 
 export const Image = Immutable.Record({
-  type: ':square',
-  height: 408,
-  width: 408,
+  type: null,
+  height: null,
+  width: null,
   url: null,
 });
 
-export const ImageRefs = Immutable.Record({
-  square: new Image(),
-  square2x: new Image(),
+const ImageRefs = Immutable.Record({
+  square: null,
+  square2x: null,
+  medium: null,
+  small: null,
+  thumb: null,
+});
+
+export const ResponsiveImage = Immutable.Record({
```

Comment:
Alternatively, ResponsiveAvatarImage & ResponsiveListingImage could be separated since they have different crop sizes in the data source (i.e. R&A).